lib/findProjectRoot.js

```diff
@@ -2,7 +2,7 @@ import fs from 'fs';
 import path from 'path';
 
 function findRecursive(directory) {
-  if (directory === '/') {
+  if (directory === '/' || /^[A-Z]:\\$/.test(directory)) {
```

Comment:
For marginally better performance, you could initialize this regex in a constant at the top of the file.